edb/server/pgcon/pgcon.pyx

```diff
@@ -289,8 +305,26 @@ async def connect(connargs, dbname, tenant_id):
             ignore_data=True,
         )
 
-    if INIT_CON_SCRIPT is None:
-        INIT_CON_SCRIPT = _build_init_con_script()
+    if 'in_hot_standby' in pgcon.parameter_status:
+        # in_hot_standby is present in Postgres 14 and above
```

Comment:
Is it *always* present?

```suggestion
        # in_hot_standby is always present in Postgres 14 and above
```

Reply:
Yes according to my test, I tried to read the Postgres GUC code but it may take too much time.